src/openrct2/localisation/Formatting.h

```diff
@@ -0,0 +1,102 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2020 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../common.h"
+#include "Language.h"
+
+#include <sstream>
+#include <string>
+#include <string_view>
+#include <utility>
+
+namespace OpenRCT2
+{
```

Comment:
Maybe there should be one more namesape? `OpenRCT2::Formatting` or `OpenRCT2::Localisation`...